unidist/core/backends/mpi/core/serialization.py

```diff
@@ -21,6 +22,15 @@
 import msgpack
 import gc  # msgpack optimization
 
+# Pickle 5 protocol compatible types check
+compatible_modules = ("pandas", "numpy")
+available_modules = {}
+for mod_name in compatible_modules:
+    try:
+        available_modules[mod_name] = importlib.import_module(mod_name)
+    except ModuleNotFoundError:
+        pass
```

Comment:
You already have module names in dunder-property `__name__`. This can be simplified as follows:
```suggestion
available_modules = []
for module_name in compatible_modules:
    try:
        available_modules.append(importlib.import_module(module_name))
    except ModuleNotFoundError:
        pass
```